src/integrationTest/java/org/radarcns/integration/util/ExpectedDataSetFactory.java

```diff
@@ -158,7 +145,7 @@ public EffectiveTimeFrame getEffectiveTimeFrame(Long value) {
             DoubleArrayCollector dac = (DoubleArrayCollector) expectedValue.getSeries().get(key);
 
             switch (sensor) {
-                case "ACCELEROMETER":
+                case "EMPATICA_E4_v1_ACCELEROMETER":
```

Comment:
`ACCELEROMETER` was made to be device-agnostic. I'd prefer that over adding the device to the type of sensor.